naiserator.go

```
@@ -160,6 +164,22 @@ func (n *Naiserator) createOrUpdateMany(resources []runtime.Object) error {
 	return result.ErrorOrNil()
 }
 
+func (n *Naiserator) removeOrphanIngresses(logger *log.Entry, app *v1alpha1.Application) error {
+	if len(app.Spec.Ingresses) == 0 {
```

vurdere _fail fast_ her, altså: if len() > 0 { return nil }